spec/strings.md

```diff
@@ -0,0 +1,26 @@
+# Strings
+
+From all inspiration languages.
+
+## Syntax and semantics
+
+There SHALL be a type `string`. Strings MUST be enclosed in double quotes or single quotes.
```

Comment:
There SHALL be the types `single-string` and `double-string`. Single strings MUST be enclosed in single quotes and vice versa.